update_venv_scripts/update_courier_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+# vs_extensions = "~/miniconda3/envs/jax_rl/lib/python3.9/site-packages/launchpad/nodes/courier/"
+venv_dir = "../venv" # e.g. /home/sam/Code/ML/acme_testing/acme
+courier_utils_dir = os.path.join(venv_dir, "lib/python3.8/site-packages/launchpad/nodes/courier")
+# vs_extensions = os.path.join(venv_dir, )"~/miniconda3/envs/jax_rl/lib/python3.9/site-packages/launchpad/nodes/courier/"
```

Comment:
clean this and other folders. Maybe I should make venv_dir and python_version into CLArgs too. Yeah.